src/v3/jest.config.js

```diff
@@ -20,6 +20,8 @@ const esModules = [
 
 const devMode = process.env.NODE_ENV === 'development';
 
+const devMode = process.env.NODE_ENV === 'development';
```

Comment:
`Cannot redeclare block-scoped variable 'devMode'.ts(2451)`

Reply:
This must have happened in my last rebase, going to try again and make sure there are no additional changes. I havent touched these files so not sure why it conflicts 😓